controller/cmake/multiap-helpers.cmake

```diff
@@ -8,4 +8,10 @@ set(${PROJECT}_VERSION_STRING ${${PROJECT}_VERSION_MAJOR}.${${PROJECT}_VERSION_M
 execute_process(COMMAND "date" "+%F %T" OUTPUT_STRIP_TRAILING_WHITESPACE OUTPUT_VARIABLE ${PROJECT}_BUILD_DATE)
 message(STATUS "MultiAP ${PROJECT} Version: ${${PROJECT}_VERSION_STRING}")
 message(STATUS "MultiAP ${PROJECT} Build Date: ${${PROJECT}_BUILD_DATE}")
-message(STATUS "MultiAP ${PROJECT} Revision: ${${PROJECT}_REVISION}")
\ No newline at end of file
+message(STATUS "MultiAP ${PROJECT} Revision: ${${PROJECT}_REVISION}")
+
+find_package(MapfCommon REQUIRED)
```

Comment:
controller don't need MapfCommon

Reply:
He will need it, that's the point of this PR. Mapf common will have the encryption which will be used by the controller to encrypt the M2 TLV.